validation/views.py

```diff
@@ -49,19 +49,29 @@ def index(request):
     """
     The home page.
     """
+    is_linguist = user_is_linguist(request.user)
+
     all_class = "button button--success filter__button"
-    validated_class = "button button--success filter__button"
-    unvalidated_class = "button button--success filter__button"
+    new_class = "button button--success filter__button"
+    linked_class = "button button--success filter__button"
+    auto_validated_class = "button button--success filter__button"
```

Comment:
I should have mentioned this before, but describing the requisite classes in the `views.py` file is maybe not the best place to put them — all class based stuff should really be in the templates.

But we'll save that for a future PR! 😅

Reply:
I completely agree, I just didn't know how to do it. I think it would take a tad more time than we have now so I'll make an issue for it.